Add simplified se_block kernel (#989)

mengluy0125 opened the pull request:
Summary:

We add a helion kernel to compute 2 * x * sigmoid(x @ w)

Reviewed By: JackyJqz

Differential Revision: D84968671

oulgen reviewed (approved):
rebase please

yf225 reviewed (approved):
thanks @mengluy0125 ! might need to rebase the PR to resolve the merge conflicts

choijon5 commented:
@mengluy0125  do you mind rebasing this so that it can be properly imported to fbcode?

mengluy0125 committed:
Add simplified se_block kernel (#989)

Summary:

We add a helion kernel to compute 2 * x * sigmoid(x @ w)

Reviewed By: JackyJqz

Differential Revision: D84968671
